### PR TITLE
refactor: api requests

### PR DESCRIPTION
**Pull request by kymppi**

### CODE REVIEW

**Comment by kymppi** [comment]
Hey, added cache and refactored the bot a bit. Anyone mind giving some though on the default TTLs

File: Cargo.toml [hunk 1]
```diff
@@ -21,3 +21,5 @@ reqwest = { version = "0.11.12", features = ["blocking", "json"] }
 ureq = "2.5.0"
```
Comment (washbin) [inline Cargo.toml:21]: Let's remove this unused library, I think cargo builds everything from dependency or something

**Review by washbin — APPROVED** [review]
Looks Good to me !

**Comment by washbin** [comment]
If I recall @thenishantsapkota  is quite busy, I dont know if he will be able to review changes today, So am just gonna merge them just in case and see what I can work on next

**Comment by kymppi** [comment]
Oh yea true

**Pull request merged**